carvel/package.yaml

```diff
@@ -420,7 +420,7 @@ spec:
     spec:
       fetch:
       - imgpkgBundle:
-          image: ghcr.io/bitnami-labs/sealed-secrets-carvel@sha256:cd484bc9c0416ad1eb5048e4a7bdb33ab0300fd883e4564e1f7e4bb7c6f94318
+          image: ghcr.io/bitnami-labs/sealed-secrets-carvel:sha256-d55dd41e5221293a68abcd66cca4a4722a10b80f43d52b7e4c98cc5beac8238d.imgpkg
```

Comment:
I guess this should be just the `sha` image reference, without including the `.imgpkg` bit, @alemorcuq ?

Reply:
This is nice.